packages/radio-button/src/RadioButton.js

```diff
@@ -53,41 +52,6 @@ export default class RadioButton extends Component {
   };
 
   render() {
-    const {
-      checked: controlledChecked,
-      defaultChecked,
-      disabled,
-      label,
-      name,
-      onBlur,
-      onChange,
-      onClick,
-      onFocus,
-      required,
-      value
-    } = this.props;
-
-    return (
-      <CheckboxBehavior
-        checked={controlledChecked}
-        defaultChecked={defaultChecked}
-        onChange={onChange}
-        onClick={onClick}
-      >
-        {({ checked, handleClick }) => (
-          <RadioButtonPresenter
-            checked={checked}
-            disabled={disabled}
-            label={label}
-            name={name}
-            onBlur={onBlur}
-            onClick={handleClick}
-            onFocus={onFocus}
-            required={required}
-            value={value}
-          />
-        )}
-      </CheckboxBehavior>
-    );
+    return <RadioButtonPresenter {...this.props} />;
```

Comment:
Looks like this file is unnecessary.

Reply:
Remove it, and export RadioButtonPresenter in its place in `index.js`

Something like 
```js
export { default } from './presenters/RadioButtonPresenter';
```